homeassistant/components/sensor/miflora.py

```diff
@@ -171,5 +175,9 @@ def update(self):
             median = sorted(self.data)[int((self.median_count - 1) / 2)]
             _LOGGER.debug("Median is: %s", median)
             self._state = median
+        elif self._state is None:
+            _LOGGER.debug("State was None, ignore median and show at least "
```

Comment:
I don't think we should motivate the code in log messages. That belongs better in code comments.

Just say that we replace the old state [...] with new state [...].

Reply:
Yep, too verbose. Fixed.